server/container_create_linux.go

```diff
@@ -1076,13 +1076,6 @@ func setupSystemd(mounts []rspec.Mount, g generate.Generator) {
 		}
 		g.AddMount(systemdMnt)
 	} else {
-		systemdMnt := rspec.Mount{
-			Destination: "/sys/fs/cgroup/systemd",
-			Type:        "bind",
-			Source:      "/sys/fs/cgroup/systemd",
-			Options:     []string{"bind", "nodev", "noexec", "nosuid"},
-		}
-		g.AddMount(systemdMnt)
```

Comment:
I think we need this code chunk when running on cgroup v1 with a fresh `/sys/fs/cgroup` mount